### PR TITLE
feat(config): add feature switches to config object

### PR DESCRIPTION
**Pull request by jgroth**
Connected to Lundalogik/crm-feature#983

### Browsers tested:

Windows:
- [ ] Chrome
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/global/config.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export class Config {
     public iconPath = '';
+    public featureSwitches = {};
```
Comment (adrianschmidt) [inline src/global/config.ts]: I think this has to be typed as `any`, or the compiler will complain about properties not existing on type `Object`…

Comment (jgroth) [reply src/global/config.ts]: Oh, yeah! :smile: 
Good catch!

Comment (adrianschmidt) [reply src/global/config.ts]: Confirmed in testing, so I ⚡️'d it 😄

Comment (jgroth) [reply src/global/config.ts]: I approve! :heavy_check_mark: :smile:

**Pull request merged**